src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -3119,11 +3124,100 @@ def generate(cls, config: SubsetConfig) -> Compose:
 
         transforms = []
         for cfg_transform in config.transforms:
+            if (input_size := getattr(config, "input_size", None)) is not None:
```

Comment:
Do we need to get `input_size` every iteration?

Reply:
Thanks for pointing. I'll update it.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3759/commits/080932be38ac791a29b6f3de922fa094bf58a7f3